base/ipmicomm.patch

```diff
@@ -0,0 +1,14 @@
+From: https://github.com/slac-epics-modules/ipmiComm/pull/33/
+diff --git a/src/drvMch.c b/src/drvMch.c
+index 33e9b00..8a057d0 100644
+--- a/src/drvMch.c
++++ b/src/drvMch.c
```

Comment:
I'm missing the explanation from your upstream patch. You can always download a `git format-patch`ed commit from GitHub by appending `.patch` to its URL, as such:

https://github.com/slac-epics-modules/ipmiComm/commit/d73ee36351ac3914ed780d432e2a03c382f2c349.patch